### PR TITLE
New version: twpayne.chezmoi version 2.34.3

### PR DESCRIPTION
**Pull request by twpayne**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111215&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation failed with:
> 5:32:57 PM: C:\Users\User\AppData\Local\Microsoft\WinGet\Links\chezmoi.exe start-process : This command cannot be run due to the error: The specified executable is not a valid application for this OS platform..

File: manifests/t/twpayne/chezmoi/2.34.3/twpayne.chezmoi.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: chezmoi.exe
+  PortableCommandAlias: chezmoi
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.3/chezmoi_2.34.3_windows_arm64.zip
+  InstallerSha256: 1F888B8F731AD14D987B6B7A0B7C0313E1E263C3C0741FD32E22F468C20DFAFD
+- Architecture: x64
+  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.3/chezmoi_2.34.3_windows_arm64.zip
```
Comment (stephengillie) [inline manifests/t/twpayne/chezmoi/2.34.3/twpayne.chezmoi.installer.yaml:16]:
```suggestion
  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.3/chezmoi_2.34.3_windows_amd64.zip
```

File: manifests/t/twpayne/chezmoi/2.34.3/twpayne.chezmoi.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+PackageIdentifier: twpayne.chezmoi
+PackageVersion: 2.34.3
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: chezmoi.exe
+  PortableCommandAlias: chezmoi
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.3/chezmoi_2.34.3_windows_arm64.zip
```
Comment (stephengillie) [inline manifests/t/twpayne/chezmoi/2.34.3/twpayne.chezmoi.installer.yaml:13]:
```suggestion
  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.3/chezmoi_2.34.3_windows_i386.zip
```

File: manifests/t/twpayne/chezmoi/2.34.3/twpayne.chezmoi.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  InstallerSha256: 1F888B8F731AD14D987B6B7A0B7C0313E1E263C3C0741FD32E22F468C20DFAFD
+- Architecture: x64
+  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.3/chezmoi_2.34.3_windows_arm64.zip
+  InstallerSha256: 1F888B8F731AD14D987B6B7A0B7C0313E1E263C3C0741FD32E22F468C20DFAFD
+- Architecture: arm
+  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.3/chezmoi_2.34.3_windows_arm64.zip
```
Comment (stephengillie) [inline manifests/t/twpayne/chezmoi/2.34.3/twpayne.chezmoi.installer.yaml:19]:
```suggestion
  InstallerUrl: https://github.com/twpayne/chezmoi/releases/download/v2.34.3/chezmoi_2.34.3_windows_arm.zip
```

File: manifests/t/twpayne/chezmoi/2.34.3/twpayne.chezmoi.installer.yaml (hunk 2 above)
Comment (twpayne) [reply manifests/t/twpayne/chezmoi/2.34.3/twpayne.chezmoi.installer.yaml]: Thanks for the suggestion. The SHA256 sums also need to be updated. I've done this.

**Review by twpayne — COMMENTED** [review]
Thanks for the review. I've fixed the URLs and checksums.

In the future I'll switch to [goreleaser](https://goreleaser.com/customization/winget/) (refs https://github.com/twpayne/chezmoi/pull/3059) as this is likely more reliable than vedantmgoyal2009/winget-releaser.

**Review by mdanish-kh — APPROVED** [review]

**Comment by mdanish-kh** [comment]
@twpayne Please mark the above conversations as "resolved" to let this PR get merged through the auto-merge process

**Comment by russellbanks** [comment]
> Thanks for the review. I've fixed the URLs and checksums.
> 
> In the future I'll switch to [goreleaser](https://goreleaser.com/customization/winget/) (refs [twpayne/chezmoi#3059](https://github.com/twpayne/chezmoi/pull/3059)) as this is likely more reliable than vedantmgoyal2009/winget-releaser.

Sorry about the recent issues @twpayne. I'm a sole developer on Komac so edge cases or odd issues are hard to spot on my own.

**Comment by twpayne** [comment]
> Sorry about the recent issues @twpayne. I'm a sole developer on Komac so edge cases or odd issues are hard to spot on my own.

As I understand it, you fixed Komac quickly. I'm more unsure of vedantmgoyal2009/winget-releaser as winget-releaser seems to lack tests and a release process.
